content/applications/inventory_and_mrp/purchase/manage_deals/manage.rst

```diff
@@ -156,9 +152,9 @@ Finally, click the :guilabel:`Create Bill` button to create a bill for the purch
    :guilabel:`User Error` pop-up to appear. The :guilabel:`Purchase Order` requires the receipt of
    at least partial quantity of the items included on the order to create a vendor bill.
 
-.. image:: manage/manage-user-error-popup.png
-   :align: center
-   :alt: User error pop-up for received quantities control policy.
+   .. image:: manage/manage-user-error-popup.png
```

Comment:
making the comment here since I can't comment on that line, but in lines 52-53 the instruction says
> Then, check the box next to :guilabel:`3-way matching: purchases, receptions, and bills`,

but I think it's just "3-way matching" now
![image](https://github.com/odoo/documentation/assets/23611711/7e55574a-3cd3-479d-af5c-c25bd27903dc)

Reply:
You're right, nice catch! Suggestion implemented ✅